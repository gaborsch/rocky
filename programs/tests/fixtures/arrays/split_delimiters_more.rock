letters = "ABC"

split letters into middle with "B" ( -> "A", "C" )
say middle at 0
say middle at 1
say middle at 2

split letters into start with "A" ( -> "", "BC" )
say start at 0
say start at 1
say start at 2

split letters into end with "C" ( -> "AB" ) ( expected "AB", "")
say end at 0
say end at 1
say end at 2

split letters into nosep with ""
say nosep at 0
say nosep at 1
say nosep at 2
say nosep at 3
