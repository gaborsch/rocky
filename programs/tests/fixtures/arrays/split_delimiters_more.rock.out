A
C
mysterious

BC
mysterious
AB

mysterious
A
B
C
mysterious
